lib/handleRequest.js

```diff
@@ -125,29 +128,42 @@ function validationCompleted (request, reply, validationErr) {
 function preHandlerCallback (err, request, reply) {
   if (reply.sent) return
 
+  const store = { request, reply }
+
   if (err != null) {
     reply[kReplyIsError] = true
     reply.send(err)
+    store.error = err
+    channels.end.publish(store)
```

Comment:
I assume this should be `error`?

```suggestion
    channels.error.publish(store)
```